### ISSUE TITLE
project dependices package version incompatible

### ISSUE THREAD

**Issue by PandyYang** [my8100/scrapydweb#202]
not a bug, just base on my current env, give someone some suggestions...
follow the step by readme.can't start the app immediately, because
download the latest packge is incompatible, so it can't run, must update some config

**Environment (please complete the following information):**
 - Operating system: mac m1
 - Python version: 3.6.15

**Additional context**
just update the requirements.txt:
```
flask==2.0.2
flask-compress==1.12
```

run:
```
 pip install -r requirements.txt
```
run:
```
 scrapydweb
```
and it should worked

note: if u have download some incompatible pkg, run :

```
pip freeze | xargs pip uninstall -y 
```
in your project virtual env, just like conda or pyenv. 
this command will delete all you have download pkg. be careful.

**Comment by afc3liuyuan** [comment]
Nice! Thanks a lot!

**Comment by heilwood** [comment]
Had same problem, thank you but i also downgraded python-werkzeug to version 2.0.3-1

**Comment by my8100** [comment]
Fixed in scrapydweb v1.4.1
